libs/ui/src/components/NeoTabs/NeoTabs.scss

```diff
@@ -0,0 +1,108 @@
+@import '../../scss/_theme.scss';
+@import '@oruga-ui/oruga/src/scss/utilities/_expressions.scss';
+@import '@oruga-ui/oruga/src/scss/utilities/_variables.scss';
+@import '@oruga-ui/oruga/src/scss/utilities/_animations.scss';
+@import '@oruga-ui/oruga/src/scss/utilities/_helpers.scss';
+@import '@oruga-ui/oruga/src/scss/components/_tabs.scss';
+
+.neo-tabs--toggle {
+  @include ktheme() {
+    background: theme('background-color');
+    box-shadow: theme('primary-shadow');
+  }
+
+  .o-tabs__content {
+    padding: 0;
+  }
+}
+
+.o-tabs {
+  &__nav {
+    overflow: auto;
+    white-space: nowrap;
+
+    &--toggle {
+      padding-bottom: 0;
+
+      & + .o-tabs__content {
+        @include ktheme() {
+          border: 1px solid theme('border-color');
+          border-top: 0;
+        }
+      }
+    }
+
+    &-item {
+      font-size: 1rem;
+
+      &-default {
+        @include ktheme() {
+          color: theme('text-color');
+
+          &--active {
+            border-bottom-color: theme('k-primary');
+            color: theme('k-primary');
+          }
+
+          &:hover:not(.o-tabs__nav-item-default--active) {
+            background-color: unset;
+            border-bottom-color: theme('k-redaccent');
+          }
+        }
+      }
+
+      &-toggle {
+        padding: 8px;
```

Comment:
Imo this can be changed with oruga class

Reply:
thanks, one left. not quite understand this one, any example that I can follow?

Reply:
oh, will remove it then